### PR TITLE
RAC-1984 fixed flawed CIT test. Should expect 404 instead of []

### PR DESCRIPTION
**Pull request by lacarb**
This is the new CIT test for https://github.com/RackHD/on-http/pull/551

The api this CIT test is testing was broken and always returned [] no matter what it was given as input. The above PR corrects for this. The Cit test that this PR changes was built expecting [] which is wrong. The test now expects a 404 response when provided a bad id.

### CODE REVIEW

**Comment by lacarb** [comment]
test this please

File: test/tests/api/v2_0/nodes_tests.py [hunk 1]
```diff
@@ -296,9 +296,11 @@ def test_node_workflows_get(self):
         for resp in resps:
             assert_not_equal(0, len(resp), message='No Workflows found for Node')
 
-    	Api().nodes_get_workflow_by_id('fooey')
-        resps_fooey = self.__get_data()
-    	assert_equal(len(resps_fooey), 0, message='Should be empty')
+        try:
+            Api().nodes_get_workflow_by_id('fooey')
+            fail(message='did not raise exception')
```
Comment (johren) [inline test/tests/api/v2_0/nodes_tests.py]: Is it possible to indicate "what" did not raise the exception?   I'm just trying to imagine what information I will have if this fails to point me to what failed.

File: test/tests/api/v2_0/nodes_tests.py [hunk 2]
```diff
@@ -296,9 +296,11 @@ def test_node_workflows_get(self):
...
+        except rest.ApiException as e:
+            assert_equal(404, e.status, message='unexpected response {0}, expected 404'.format(e.status))
```
Comment (johren) [inline test/tests/api/v2_0/nodes_tests.py]: Same here.  Is there something printed above that tells me which API was called that gave me the unexpected response?

**Comment by lacarb** [comment]
Jenkins: test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
http://rackhdci.lss.emc.com/job/Templates/job/SmokeTest-CIT/413/ 
one test case error ```test_node_workflows_post.test_node_workflows_post```

**Comment by changev** [comment]
test this please

**Comment by panpan0000** [comment]
test this please

**Comment by pscharla** [comment]
jenkins: test this please

**Comment by lacarb** [comment]
Please hold off on merging this for the time being.

**Comment by lacarb** [comment]
This PR should be green when this ticket is merged
https://github.com/RackHD/on-http/pull/551
Please show it some love when the pipeline is cleared :)

**Comment by lacarb** [comment]
test this please

**Comment by lacarb** [comment]
@RackHD/corecommitters 
Please merge this when you have the chance.

**Comment by PengTian0** [comment]
test this please

**Comment by johren** [comment]
CIT tests are all being converted to FIT framework.  Therefore, this has become obsolete and the originator is no longer active on the project.  Please recreate as a FIT test.

**Pull request closed without merging**